app/views/hyku/api/v1/work/_work.json.jbuilder

```diff
@@ -33,14 +33,20 @@ json.cache! [@account, :works, work.id, work.solr_document[:_version_], work.mem
   json.date_submitted work.date_uploaded
   json.degree work.try(:solr_document)&.to_h&.dig('degree_tesim')
   json.dewey work.try(:solr_document)&.to_h&.dig('dewey_tesim')
+
   json.doi work.try(:doi)
+  if (doi = work.try(:solr_document)&.to_h&.dig('doi_ssi')).present?
+    json.solr_doi "https://doi.org/#{doi}"
```

Comment:
This is changing the name of the field on the front end so can we keep it as `json.doi` to avoid any extra work for Rees